gno.land/pkg/integration/defaultsigner.go

```diff
@@ -0,0 +1,32 @@
+package integration
```

Comment:
Why not name this file `signer.go`?

Reply:
renamed on 2a67769b4f165ffd4d9ed62cf27891387fa8b949